dual_net.py

```diff
@@ -273,9 +277,14 @@ def eval_metrics_host_call_fn(policy_output, value_output, pi_tensor, policy_cos
             for metric_name, metric_op in metric_ops.items():
                 summary.scalar(metric_name, metric_op[1])
 
-        if est_mode == tf.estimator.ModeKeys.EVAL:
-            return metric_ops
-        return summary.all_summary_ops()
+        # Reset metrics occasionally so that they are mean of recent batches.
+        reset_op = tf.variables_initializer(tf.local_variables("mean"))
```

Comment:
This doesn't work for policy_accuracy_top1 which is tf.metric.accuarcy

I'd suggest resolving the TODO above and wrapping the metric_ops in a variable_scope("metrics") then using that scope here

Reply:
ok -- do you want to do that in this PR then?